charts/istio-alerts/templates/service-prometheusrule.yaml

```diff
@@ -1,57 +1,71 @@
-{{ $values           := .Values }}
-{{ $release          := .Release }}
-{{ if .Values.serviceRules.enabled }}
-
+{{- $values     := .Values }}
+{{- $release    := .Release }}
+{{- if .Values.serviceRules.enabled }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ .Release.Name }}-service-rules
+  name: {{ .Release.Name }}-{{ lower .Chart.Name }}-service-rules
   annotations:
     nextdoor.com/chart: {{ .Values.chart_name }}
     nextdoor.com/source: {{ .Values.chart_source }}
 spec:
   groups:
-  - name: {{ .Release.Name }}.{{ .Release.Namespace }}.serviceRules
+  - name: {{ .Release.Namespace }}.{{ .Release.Name }}.{{ .Chart.Name }}.IstioServiceRules
     rules:
-    {{ with .Values.serviceRules.http5XXMonitor }}
-    {{ if .enabled }}
-    - alert: {{ $release.Name }}-5xx-Rate-Too-High
+    {{- with .Values.serviceRules.http5XXMonitor }}
+    {{- if .enabled }}
+    - alert: 5xx-Rate-Too-High
       annotations:
-        summary: {{`High rate of 5xx errors from {{$labels.destination_service_name}} in namespace {{$labels.namespace}}`}}
+        summary: >-
+          {{`{{$labels.destination_service_name}} is throwing too many 5xx errors (namespace: {{$labels.namespace}})`}}
```

Comment:
Per our discussion, the `summary` field is where we want to have the information about the particular application and/or namespace. I've reformatted the summary to be every so slightly shorter but to make the service name front and center as the first word.